inductiva/tasks/task.py

```diff
@@ -216,19 +206,28 @@ def get_output(
         """
         self.wait()
 
-        # Check if default files are set by the output class
+        # Fetch just the first part of the method_name (e.g., "wind_tunnel")
+        method_name = self.get_info()["method_name"].split(".")[0]
+
+        # Set the default files for the output class
+        # For some scenarios, there are None default files
         filenames = None
-        if self._default_files_list and not all_files:
-            filenames = self._default_files_list
+        output_class = None
+        if method_name in output_consts.OUTPUT_CONSTS:
+            output_class = output_consts.OUTPUT_CONSTS[method_name][
+                "output_class"]
+            if not all_files:
+                filenames = output_consts.OUTPUT_CONSTS[method_name][
+                    "default_files"]
 
         output_dir = self.download_outputs(
             filenames=filenames,
             output_dir=output_dir,
             uncompress=uncompress,
             rm_downloaded_zip_archive=rm_downloaded_zip_archive)
 
-        if self._output_class:
-            return self._output_class(output_dir)
+        if output_class is not None:
```

Comment:
Does this mean that sometimes this method returns a string with a path and sometimes it returns a python object (some Scenario Output?). Is it consistent with the docstring?